src/mapeo-manager.js

```diff
@@ -149,17 +149,14 @@ export class MapeoManager extends TypedEmitter {
     this.#invite = new InviteApi({
       rpc: this.#localPeers,
       queries: {
-        isMember: async (projectId) => {
-          const projectExists = this.#db
+        isMember: (projectPublicId) =>
```

Comment:
I split this into a smaller change in #480, which can be reviewed/merged separately.